src/AuthModule/Components/Register/Register.tsx

```diff
@@ -1,7 +1,432 @@
+                    autoFocus
+                    {...register("userName", userNameValidation)}
+                  />
+                  {errors.username && (
+                    <Typography variant="body2" sx={{ color: "red" }}>
+                      {errors?.userName?.message}
+                    </Typography>
+                  )}
+                </Box>
+
+                <Box
+                  sx={{
+                    display: "flex",
+                    justifyContent: "space-between",
+                  }}
+                >
+                  <Box>
+                    <Typography sx={{ mt: 1 }}>Phone Number</Typography>
+                    <TextField
+                      // sx={{bgcolor:'#F5F6F8'}}
+                      placeholder="Please type here ..."
+                      margin="normal"
+                      required
+                      fullWidth
+                      id="phoneNumber"
+                      // label="Email Address"
+                      autoComplete="phoneNumber"
+                      autoFocus
+                      {...register("phoneNumber", phoneNumberValidation)}
+                    />
+                    {errors.phoneNumber && (
+                      <Typography variant="body2" sx={{ color: "red" }}>
+                        {errors?.phoneNumber?.message}
+                      </Typography>
+                    )}
+                  </Box>
+                  <Box>
+                    <Typography sx={{ mt: 1 }}>Country</Typography>
+                    <TextField
+                      // sx={{bgcolor:'#F5F6F8'}}
+                      placeholder="Please type here ..."
+                      margin="normal"
+                      required
+                      fullWidth
+                      id="country"
+                      // label="Email Address"
+                      autoComplete="country"
+                      autoFocus
+                      {...register("country", countryValidation)}
+                    />
+                    {errors.country && (
+                      <Typography variant="body2" sx={{ color: "red" }}>
+                        {errors?.country?.message}
+                      </Typography>
+                    )}
+                  </Box>
+                </Box>
+
+                <Box>
+                  <Typography sx={{ mt: 1 }}>Email Address</Typography>
+                  <TextField
+                    // sx={{bgcolor:'#F5F6F8'}}
+                    placeholder="Please type here ..."
+                    margin="normal"
+                    required
+                    fullWidth
+                    id="email"
+                    // label="Email Address"
+                    autoComplete="email"
+                    autoFocus
+                    {...register("email", emailValidation)}
+                  />
+                  {errors.email && (
+                    <Typography variant="body2" sx={{ color: "red" }}>
```

Comment:
Here we should use the red from the MUI pallet as well.

Reply:
It didn't work too

Reply:
I tryed to put color=red from mui pallet and it didn't work

Reply:
You can look here at the default theme and correct color names:
To get the red from the theme, you'll need to write `color="error.main"` or error.light or dark.

![image](https://github.com/SalmaAboArab/Hotel-Management-System/assets/64327685/72b29c07-0dfd-41d2-b44e-23dc285497cf)